docs/dev/news/2823.feat

```diff
@@ -0,0 +1 @@
+Introduce a new Dafny CLI UI that complies with the POSIX standard and uses verbs to distinguish between use-cases.
```

Comment:
Oh we didn't mention the new CLI already? Glad I asked then! :)

Can we say a little about how to access the new CLI? Perhaps just `Type `dafny` to view the documentation`. Otherwise it sounds like users might need to obtain and use a new binary.